app/Http/Controllers/ResourceController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\ResourceResource;
+use App\Models\Resource;
+
+class ResourceController extends Controller
+{
+    public function __invoke()
+    {
+        return ResourceResource::collection(
+            Resource::query()
+                ->with('county')
+                ->with('organisation')
+                ->with('category')
+                ->with('subcategory')
+                ->with('types')
```

Comment:
```suggestion
                ->with([
                    'county',
                    'category',
                    'subcategory',
                    'types',
                    'organisation' => fn ($query) => $query
                        ->withoutEagerLoads()
                        ->select('id', 'name'),
                ])
```

multiple eager loads pot fi pasate ca array.

```php
'organisation' => fn ($query) => $query
    ->withoutEagerLoads()               // excludem eager load-ulrile înregistrate în model
    ->select('id', 'name'),             // luăm doar id-ul și numele in db, nu toate informațiile despre organizație
```